integration-tests/mssql_test.go

```diff
@@ -245,11 +251,16 @@ var _ = Describe("MSSQL", Label("MSSQL"), func() {
 				err := broker.Update(instanceID, msSQLServiceName, customMySQLPlan["name"].(string), map[string]any{prop: value})
 
 				Expect(err).NotTo(HaveOccurred())
+				Expect(nthTerraformInvocationVars(mockTerraform, 1)).To(HaveKeyWithValue(prop, value))
 			},
 			Entry("update storage_type", "storage_type", "gp2"),
-			Entry("update iops", "iops", 1500),
+			Entry("update iops", "iops", float64(1500)),
```

Comment:
thoughts: we are mixing styles `BeNumerically()` and `float64` conversion. We may want to unify the style